src/GridFieldEditableColumns.php

```diff
@@ -130,12 +130,22 @@ public function handleSave(GridField $grid, DataObjectInterface $record)
         /** @var GridFieldOrderableRows $sortable */
         $sortable = $grid->getConfig()->getComponentByType(GridFieldOrderableRows::class);
 
+        // Fetch the items before processing them
+        $ids = array_keys($value[self::POST_KEY]);
```

Comment:
```suggestion
        $ids = array_keys($value[self::POST_KEY]);
        if (empty($ids) {
            return;
        }
```
Filtering by an array of values will throw an exception when the array is empty, limitation of the ORM